pkg/config/config.go

```diff
@@ -59,10 +64,12 @@ type Config struct {
 
 	// CPU costs for various ingress types
 	CPUCost CPUCostConfig `yaml:"cpu_cost"`
+}
 
+type InternalConfig struct {
 	// internal
-	ServiceName string `yaml:"-"`
-	NodeID      string // Do not provide, will be overwritten
+	ServiceName string `yaml:"service_name"`
+	NodeID      string `yaml:"node_id"` // Do not provide, will be overwritten
```

Comment:
since we'll override it, should it be `yaml:"-"` ? same for ServiceName

Reply:
This needs to be serialized when passing the config to the handler process